backend/FwHeadless/ProjectLookupService.cs

```diff
@@ -13,4 +14,9 @@ public class ProjectLookupService(LexBoxDbContext dbContext)
             .FirstOrDefaultAsync();
         return projectCode;
     }
+
+    public async Task<bool> IsCrdtProject(Guid projectId)
+    {
+        return await dbContext.Set<ServerCommit>().AnyAsync(c => c.ProjectId == projectId);
```

Comment:
It seems weird that we need to do it this way i.e. we don't actually have a list of projects.
We have `ProjectData` in `LcmCrdtDbContext`, but I guess that's only on clients?
So, we have metadata/a list of projects, but that data doesn't get synced to the server, right?

Reply:
ProjectData is client side only correct. I mean there is our lexbox projects, however right now the only way to know if it is a CRDT project is to see if there's any commits, which is what this is doing.